tests/Homely.AspNetCore.Mvc.Helpers.Tests/TestControllerTests/PostTests.cs

```diff
@@ -41,7 +41,7 @@ public async Task GivenPostingAValidModel_Post_ReturnsAndHttp201()
             // Assert.
             response.EnsureSuccessStatusCode();
             response.StatusCode.ShouldBe(HttpStatusCode.Created);
-            response.Headers.Location.ShouldBe(new Uri("http://localhost/test/7"));
+            response.Headers.Location.ShouldBe(new Uri("http://localhost/test/9"));

```

Comment:
magic value?

Reply:
yep. i'll re-fix.

🔧

Reply:
sorry, i meant.. _why_ is it a magic value? 

i had a look through the code, and eventually figured it out to be because the repo is setting the id with a counter, so i see now.

would prefer to have it more understandable, e.g in the test we should be setting the callback there to set the id, so we can control and assert the id in the test itself instead of it being set magically 'somewhere else', but.. that's out of the scope of this PR... so fine as is. if you agree i can open a GH issue and we can fix it up later?

Reply:
EDIT: just reviewing the code now.. and i see you're doing what i want/prefer [here](https://github.com/Homely/Homely.AspNetCore.Mvc.Helpers/pull/30/files#diff-ee170d34e0e22288fc4d73073ca034a4R49)

can we change this code to be like that?

Reply:
I agree and am confused... because my code doesn't have magic numbers in it (any more) i fixed it .. but .. GH isn't updated?

Reply:
![image](https://user-images.githubusercontent.com/899878/90458536-023f1080-e142-11ea-9e27-a2510b7d34cf.png)

AH! so it was showing the old code and we were both still 'looking' at that 😊 

if we go to the FILES CHANGED, we can see the latest version and it's good.

can you check please @RPM1984 ?

Reply:
(yes, i know u approved already)

Reply:
yep, the commit you just pushed up has the correct code. ta.

Reply:
LGTM